articles/connections/passwordless/sms-gateway.md

```diff
@@ -1,66 +1,74 @@
 
-## Configure your SMS Gateway
+2. Obtain information about the Passwordless SMS connection.
 
-You will need to configure your own SMS Gateway through the Management API. 
+    To do this, make a GET call to the [Connections](/api/management/v2#!/Connections/get_connections) endpoint. The response from the endpoint will be an array of objects. Each object represents one connection affiliated with your tenant.
+    
+    Find the connection used for your Passwordless SMS connection (you can search for the `"name": "sms"` property). Notice that the connection currently displays the Twilio information you provided during the setup process.
 
-1. Get your connections (strategy: `sms`) from the [GET connections endpoint](/api/v2#!/Connections/get_connections).
-
-This will return your SMS connection with your Twilio settings:
-
-```
+```json
 [
-  {
-    "id": "con_...",
-    "options": {
-      "strategy": "sms",
-      "twilio_sid": "...",
-      "twilio_token": "...",
-      "from": "+1 234 567",
-      "template": "Your verification code is: @@password@@",
-      "brute_force_protection": true,
-      "disable_signup": false,
-      "name": "sms",
-      "forward_req_info": "true",
-      "syntax": "md_with_macros",
-      "totp": {
-        "time_step": 300,
-        "length": 6
-      }
-    },
-    "strategy": "sms",
-    "name": "sms",
-    "enabled_clients": [
-      ...
-    ]
-  }
+    {
+        "id": "con_UX85K7K0N86INi9U",
+        "options": {
+            "disable_signup": false,
+            "name": "sms",
+            "twilio_sid": "TWILIO_SID",
+            "twilio_token": "TWILIO_AUTH_TOKEN",
+            "from": "+15555555555",
+            "syntax": "md_with_macros",
+            "template": "Your SMS verification code is: @@password@@",
+            "totp": {
+                "time_step": 300,
+                "length": 6
+            },
+            "messaging_service_sid": null,
+            "brute_force_protection": true
+        },
+        "strategy": "sms",
+        "name": "sms",
+        "is_domain_connection": false,
+        "realms": [
+            "sms"
+        ],
+        "enabled_clients": []
+    }
 ]
 ```
 
-2. Modify the options of the connection:
+     **Make anote of the `id` value of this connection, since you will need it to update your connection with your SMS Gateway information.**
 
- - `provider`: Set this to `sms_gateway`
- - `gateway_url`: Set this to the URL of your SMS Gateway. Note that Auth0 must be able to reach it for this to work.
+3. Update the connection. You can do this by making a PATCH call to the [Update a Connection](/api/management/v2#!/Connections/patch_connections_by_id) endpoint. More specifically, you'll be updating the connections `options` object to provide information about the SMS Gateway.
 
-```
+    **You must send the entire `options` object with each call, otherwise, your new payload will overwrite the existing data.**
```

Comment:
Avoid the word payload? Good for tech users, maybe not so much for people who haven't studied JWTs?